.travis.yml

```diff
@@ -12,7 +12,7 @@ rvm:
   - ruby-head
 env:
   global:
-    - JRUBY_OPTS='--debug -J-Xmx1000M' # get more accurate coverage data in JRuby
+    - JRUBY_OPTS='-J-Xmx1000M'
```

Comment:
I believe that is still the case (that JRuby needs `--debug` for more accurate coverage data), but since code coverage is no longer generated on Travis, `--debug` can be removed from this config file.

`--debug` are present in the CircleCI JRuby configuration.

Reply:
Aaaaah, of course! :+1: